src/styles/styles.js

```diff
@@ -1350,6 +1350,12 @@ const styles = {
         width: '100%',
     },
 
+    iouPreviewBoxLoading: {
+        minHeight: 66,
```

Comment:
@Julesssss wasn't totally sure how to match the height of the IOU preview box aside from hardcoding the base height where the names don't spill over to three lines. Let me know if you had any thoughts on how to handle the case where we do wrap to 3 lines... maybe truncate the names + add an elipsis? 


https://user-images.githubusercontent.com/16747822/119170789-9cf1b600-ba18-11eb-801b-8956d808ce2f.mov

![image](https://user-images.githubusercontent.com/16747822/119170695-7a5f9d00-ba18-11eb-8399-cc53c3a942de.png)

![image](https://user-images.githubusercontent.com/16747822/119170767-96fbd500-ba18-11eb-9832-6ad57424888a.png)

Reply:
Oh damn, that's a great point... I wonder if we could predict the number of lines? But that's a lot of work for not much gain. 

I wonder what @shawnborton thinks about the resizing, is it something we can live with?

Reply:
Maybe if the View animated while extending it's height it wouldn't look so bad, but I don't think that should be a blocker personally.

Reply:
Couple of things here:
- I'm surprised that the "XXXX paid XXXX" line doesn't extend the full width of the IOU preview component. Why is that? ![image](https://user-images.githubusercontent.com/2319350/119175608-e0e7b980-ba1e-11eb-8686-c45cb6b35b36.png)
- Any reason why we aren't just using first names for the "XXXX paid XXXX" line?
- All of that is to say is that I feel like we should optimize for the "XXXX paid XXXX" line just being one line. So we could use that as the minHeight of the preview component, and then I think it's fine if we need to animate the preview becoming a bit larger should that "XXXX paid XXXX" line end up being two lines. 

Let me know if all of that makes sense!

Reply:
>Any reason why we aren't just using first names for the "XXXX paid XXXX" line?

I'd be in favour of dropping the secondName personally.

Reply:
I can modify the line to go the full length of the box, I think it's just nested in a `<View>` that has a width restriction. Also happy to swap to first names only, though might be confusing if two users have the same name? `Tom paid Tom`?

Reply:
I agree it might be confusing for those edge cases, but they feel like edge cases to me. I would be down to try using first names and seeing if it becomes a problem first, and then maybe we can solve by doing something like Tom J paid Tom S or something like that?

Reply:
How does this look? 
![image](https://user-images.githubusercontent.com/16747822/119185565-b18b7980-ba2b-11eb-9611-05f3dfbf40fd.png)
![image](https://user-images.githubusercontent.com/16747822/119185572-b3553d00-ba2b-11eb-9acc-64b0ba6654bc.png)

Reply:
I don't think we have any logic for truncating last names to the first letter but I'm sure we could add something to do that if we find it's a problem.

Reply:
Yeah exactly, let's see if it's even a problem first. Otherwise this looks great!